Resources/doc/index.rst

```diff
@@ -348,5 +348,21 @@ class (as shown above), you can access it in the ``load()`` method.
     If you are too lazy to implement the needed method :method:`Symfony\\Component\\DependencyInjection\\ContainerInterface::setContainer`,
     you can then extend your class with :class:`Symfony\\Component\\DependencyInjection\\ContainerAware`.
 
+Automatically Generated Fixtures with Hautelook/AliceBundle
+-----------------------------------------------------------
+
+You can also use the popular `hautelook/AliceBundle`_ bundle to declare your fixtures:
+
+.. code-block:: yaml
+
```

Comment:
this should have a commented out sample path to the file to know where this needs to be placed